packages/hosted-credit-card-integration/project.json

```diff
@@ -19,5 +19,5 @@
       }
     }
   },
-  "tags": ["scope:integration"]
+  "tags": ["scope:shared"]
```

Comment:
should we do like this?
```suggestion
  "tags": ["scope:integration", "scope:shared"]
```

Reply:
I thought we were supposed to use only 1 tag 🤔